### ISSUE TITLE
Cache cleaner

### ISSUE THREAD

**Issue by bonnebulle** [minbrowser/min#1961]
A great feature should be... a button to easly clean the cache !

**Comment by PalmerAL** [comment]
The "history > clear all history" option will clear the cache as well. Or do you want a way to clear the cache but keep history?

**Comment by bonnebulle** [comment]
Yep, exatly, a dedicated button/option for clean/remove the cache only ! (Ans Keep histoy) 

If it is not possible I will remove the two in once
 
Thanks for all
